arithmetization/src/test/java/net/consensys/linea/zktracer/module/precompileLimits/ModexpIllegalOperationTests.java

```diff
@@ -13,7 +13,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package net.consensys.linea.zktracer.containers;
+package net.consensys.linea.zktracer.module.precompileLimits;
```

Comment:
Will this change be ok if we apply it on top of the old release `beta-v1.2.0-rc3` ? Which doesn't for instance contain the changes related to `transactionBundles` ?